src/gmt_support.c

```diff
@@ -14583,6 +14583,14 @@ char *gmt_putusername (struct GMT_CTRL *GMT) {
 	struct passwd *pw = NULL;
 	pw = getpwuid (getuid ());
 	if (pw) return (pw->pw_name);
+#endif
+#ifdef WIN32
+	{
+		char name[GMT_LEN256] = {""};
+		DWORD Size = _tcslen (name);
+		if (GetUserName (name, &Size))
+			return (name);
+	}
 #endif
```

Comment:
Compilation errors:
```
..\src\gmt_support.c(14590): error C2065: 'DWORD': undeclared identifier
..\src\gmt_support.c(14590): error C2146: syntax error: missing ';' before identifier 'Size'
..\src\gmt_support.c(14590): error C2065: 'Size': undeclared identifier
..\src\gmt_support.c(14590): warning C4013: '_tcslen' undefined; assuming extern returning int
..\src\gmt_support.c(14591): warning C4013: 'GetUserName' undefined; assuming extern returning int
..\src\gmt_support.c(14591): error C2065: 'Size': undeclared identifier
```

Reply:
Yikes, I thought Windows.h was included elsewhere but was not... Please try again.